frontend-react/yarn.lock

```diff
@@ -50,37 +50,14 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@babel/compat-data@npm:^7.22.6, @babel/compat-data@npm:^7.23.3, @babel/compat-data@npm:^7.23.5":
+"@babel/compat-data@npm:^7.22.6, @babel/compat-data@npm:^7.23.5":
```

Comment:
Not sure we should be updating this file as part of this PR?

Reply:
This looks like a deduplicate, and often occurs after dependabots have been merged since dependabot does not perform our pre-commit hooks.

Reply:
@snesm for future reference, should yarn.lock be left off PRs?

Reply:
@JFisk42 we use yarn.lock to sync all the developers with the same dependency versions that will match production.  We need to keep it. But if your PR does not touch any frontend sources, you could leave it for someone else to update.